docs/UserGuide.md

```diff
@@ -187,7 +186,7 @@ Examples:
 
 Screenshot: 
 ![Friend edit Screenshot](images/ui-screenshots/friend-edit.png)
-<br><center><ins>Image: gitGud after editing an existing friend `kevcool`'s name to `Kevin Foong`</ins></center>
+<br><center><ins>Image: Command feedback box after editing an existing friend `kevcool`'s name to `Kevin Foong`</ins></center>
```

Comment:
Need to add 2 spaces/enter key after "Screenshot:" for new line
![image](https://user-images.githubusercontent.com/50147457/140608911-3f382e96-57b7-41a7-9b72-096c14b4a91d.png)

Reply:
Added.